YHP_website/src/main/java/models/Person.java

```diff
@@ -0,0 +1,57 @@
+package models;
+
+import java.util.UUID;
+
+public abstract class Person
+{
+	private final UUID id;
+	private final String forename;
+	private final String surname;
+
+	// @SuppressWarnings("unused")
+	// private Person()
+	// {
+	// //Default constructor should not be used
+	// this.id = null;
+	// this.forename = null;
+	// this.surname = null;
+	// }
+
+	public Person(String forename, String surname)
+	{
+		this.id = UUID.randomUUID();
+		this.forename = forename;
+		this.surname = surname;
+	}
+
+	public Person(String uuid, String forename, String surname)
```

Comment:
change  uuid type to `UUID` as database should be able to store values nativley as UUIDs